src/renderer/viz/expressions/Zoomrange.js

```diff
@@ -26,24 +26,25 @@ import { implicitCast, checkType, checkExpression } from './utils';
  * @api
  */
 export default class Zoomrange extends BaseExpression {
-    constructor (zoomBreakpointList) {
-        zoomBreakpointList = implicitCast(zoomBreakpointList);
+    constructor (_zoomBreakpointList) {
```

Comment:
Shouldn't the ctor param be named without underscore?

Reply:
You're right, this is a mistake when replacing the string.